src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -170,7 +171,7 @@ async function getAndDispatchFeatures(to, featuresPromise, store) {
             if (!query.z) {
                 await store.dispatch('zoomToExtent', {
                     extent: extent,
-                    maxZoom: 8,
+                    maxZoom: STANDARD_ZOOM_LEVEL_1_25000_MAP,
```

Comment:
The use of this variable is valid only (and ONLY) if the current projection system is a world-wide projection (Mercator).
This is not a LV95 zoom!

Reply:
the maxZoom parameter, when given to `zoomToExtent` goes through `transformStandardZoomLevelToCustom`, which was not the case when the parameter was first introduced. (see https://github.com/geoadmin/web-mapviewer/blob/0f3a4dbb104ae68a14f2aa4881104be47507c173/src/store/modules/position.store.js#L269-L273 )
it assumes the maxZoom level you give to the function is not a LV95 zoom.

If I give the `8` from before (which is the LV95 zoom level I would like) to the function, it transforms it to zoom level 2 because it thinks it's a world wide projection level 8 zoom and that it needs to be converted to LV95

Reply:
Then I'd say the issue is that we reprocess the given `maxZoom`, as if it was a Mercator zoom.

The logic should maybe be changed to :
 - if `maxZoom` is defined, use it as it is
 - if not set, use `STANDARD_ZOOM_LEVEL_1_25000_MAP`
   - Are we in a custom coordinate system? If yes, transform `STANDARD_ZOOM_LEVEL_1_25000_MAP` to match the current coordinate system

This way `maxZoom` can be set to 8, which is correct in the context of LV95.

It might be good to have new variable on each projection, called something like "default detail zoom level` or "zoom level 1:25000", in order to reduce the amount of magic numbers in our code

Reply:
I like this solution.